ruleset/sca/windows/cis_win10_enterprise.yml

```
@@ -351,13 +361,13 @@ checks:
     remediation: "To establish the recommended configuration via GP, set the following UI path to Disabled: Computer Configuration\\Policies\\Windows Settings\\Security Settings\\Local Policies\\Security Options\\Domain member: Disable machine account password changes."
     compliance:
       - cis: ["2.3.6.4"]
-      - cis_csc: ["16"]
       - pci_dss: ["8.2.4"]
       - tsc: ["CC6.1"]
     references:
-      - 'CCE-37508-9'
-    condition: all
+      - "CCE-37508-9"
+    condition: any
     rules:
+      - 'r:HKEY_LOCAL_MACHINE\System\CurrentControlSet\Services\Netlogon\Parameters -> DisablePasswordChange'
```

This is disabled by default. I think this rule should be not:
```suggestion
      - 'not r:HKEY_LOCAL_MACHINE\System\CurrentControlSet\Services\Netlogon\Parameters -> DisablePasswordChange'
```

---

Done 2a00c95